content/web/examples.md

```diff
@@ -7,6 +7,69 @@ draft: false
 ## Examples
 
 The full code for the examples below is available in the examples module of the Web WorldWind project.
+<<<<<<< HEAD
```

Comment:
The content looks good, but it looks like Git merge strings are still in the source.